CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 2018-02-15
+
+## Summary
+
+This is an awesome release!
```

Comment:
Aren't all releases? :)

Reply:
@ekohl Is https://github.com/voxpupuli/modulesync/pull/142 the most significant change?  I really think we should have something more sensible the 'this is an awesome release!'

Reply:
I just wanted to be funny :P